src/documents/components/ListFilterBar/ListFilterBar.js

```diff
@@ -88,95 +103,319 @@ const ListFilterBar = ({fileList, filteredList}) => {
 
   return (
     <>
-      <Responsive
-        as={Segment}
-        maxWidth={699}
-        basic
-        className="noHorizontalPadding"
-      >
-        <Button
-          data-testid="show-sort-button"
-          basic={!showSort}
-          floated="right"
-          primary={sortMethod !== ''}
-          icon="sort"
-          onClick={() => setShowSort(!showSort)}
-        />
-        <Button
-          data-testid="show-filter-button"
-          basic={!showFilter}
-          floated="right"
-          primary={typeFilterStatus !== '' || tagFilterStatus !== ''}
-          icon="filter"
-          onClick={() => setShowFilter(!showFilter)}
-        />
-        <Input
-          aria-label="file-search-input"
-          className="mr-80"
-          fluid
-          icon="search"
-          onChange={(e, {value}) => {
-            setSearchString(value);
-          }}
-          value={searchString}
-        />
-        {showFilter && (
-          <Segment>
-            <p className="mb-5">
-              <Icon name="filter" />
-              Filter by:
-            </p>
-            <Dropdown
+      {selection.length > 0 ? (
```

Comment:
Would it be better to make the two states their own components within the filter bar?